ddtrace/api.py

```diff
@@ -169,14 +171,12 @@ def send_traces(self, traces):
     def send_services(self, *args, **kwargs):
         return
 
-    def _put(self, endpoint, data, count=0):
```

Comment:
Context, `count` used to be optional since we used to use the same endpoint for sending services, which did not include the trace count header.

Reply:
🙇